core/CMakeLists.txt

```diff
@@ -80,13 +80,13 @@ endif ()
 add_asm_target(arch/pre_inject_asm.asm preinject_asm_src preinject_asm_tgt ""
   "-DNOT_DYNAMORIO_CORE_PROPER" "${asm_deps}")
 
-if (UNIX AND X86)
```

Comment:
Just a reminder that there is an outstanding bug on the X86 macro definition causing conflicts in google3.